src/types/Credential.ts

```diff
@@ -5,29 +5,49 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-import { DidUri } from '@kiltprotocol/types'
-import { VerifiableCredential, constants } from '@kiltprotocol/vc-export'
+import type { DidUri as Did } from '@kiltprotocol/types'
```

Comment:
Why cast it is `Did`?

Reply:
It doesn't feel consistent

Reply:
in the new sdk this type is simply called `Did`, and because I back-ported the code from a branch that uses the new sdk I simply renamed it accordingly. But I could also change all occurrences to `DidUri` I guess

Reply:
db79d6a43283cc627427a6cffe4789d7d8eb25f0